timesketch/ui/templates/sketch/explore.html

```diff
@@ -0,0 +1,154 @@
+          <option disabled selected>Choose view</option>
+          {% for view in sketch.get_named_views %}
+          <option value="{{ url_for('sketch_views.explore', sketch_id=sketch.id) }}view/{{ view.id }}/">{{ view.name }}</option>
+          {% endfor %}
+        </select>
+        {% endif %}
+        <br>
+        <br>
+      </div>
+    </div>
+
+    <!-- Filter card -->
+    <div class="row">
+      <div class="col-md-12">
+        <div ng-show="showFilters" class="card filter-card">
+          <h5>Time range</h5>
+          <form class="form-inline" role="form">
+            <div class="form-group">
+              <input ng-model="filter.time_start" type="text" class="form-control timerange-input" placeholder="Start 2014-01-01T23:00:00">
+            </div>
+            <i style="margin-left:5px;margin-right:5px;" class="fa fa-arrow-right"></i>
+            <div class="form-group">
+              <input ng-model="filter.time_end" type="text" class="form-control timerange-input" placeholder="End 2014-01-01T23:00:00">
+            </div>
+            <br><br>
+            <input style="margin-right:5px;" type="submit" class="btn btn-primary" value="Apply filter" ng-click="search()">
+            <input type="reset" class="btn btn-default" value="Clear" ng-click="clearFilter()">
+          </form>
+        </div>
+      </div>
+    </div>
+
+    <!-- Timeline picker card -->
+    <div class="row">
+      <div class="col-md-12">
+        <div class="card timeline-picker-card">
+          <div class="btn-group">
+            <button class="btn btn-default" filter="filter" search="search" meta="meta" index-enabler><i class="fa fa-check"></i> Enable all</button>
+            <button class="btn btn-default" filter="filter" search="search" meta="meta" index-disabler><i class="fa fa-ban"></i> Disable all</button>
+          </div>
+          <br>
+          <br>
+          {% for timeline in sketch.timelines %}
+          <div filter="filter" search="search" search_starred="search_starred" meta="meta" index-chooser index="{{ timeline.searchindex.index_name }}" style="background:#f7f7f7;padding: 10px;margin-right: 10px;cursor: pointer;margin-bottom: 5px;" class="pull-left timelinebox">
+            <div class="color-box pull-left" style="background:#{{ timeline.color }};"></div>
+            <div class="timeline-title pull-left">
+              {{ timeline.name }}
+            </div>
+            <div class="timeline-checkbox pull-right"><input type="checkbox"></div>
+          </div>
+          {% endfor %}
+        </div>
+      </div>
+    </div>
+
+    <!-- Result card -->
+    <div class="row">
+      <div class="col-md-12">
+        <div class="card">
+          {% raw %}
+          <div ng-show="meta" style="padding:10px;">
+            {{ meta.es_total_count }} events ({{ meta.es_time/1000 }}s)
+          </div>
+          <div ng-repeat="event in events" ng-controller="EventCtrl">
```

Comment:
Absolutely. I have added a TODO to fix this. In this PR I want to move the stuff in and get comments and implement the changes in a future PR.